DataFormats/BTauReco/interface/TemplatedSoftLeptonTagInfo.h

```diff
@@ -25,7 +25,8 @@ class SoftLeptonProperties {
         deltaR(   std::numeric_limits<float>::quiet_NaN() ),
         ratio(    std::numeric_limits<float>::quiet_NaN() ),
         ratioRel( std::numeric_limits<float>::quiet_NaN() ),
-        elec_mva( std::numeric_limits<float>::quiet_NaN() )
+        elec_mva( std::numeric_limits<float>::quiet_NaN() ),
+        charge(   std::numeric_limits<int>::quiet_NaN() )
```

Comment:
lepton charge=0 is accepted by the tagger and treated as a jet without leptons, so it could still produce a valid result, if any other TagInfo contains information.